src/assets/stylesheets/utilities/_break_points.scss

```diff
@@ -42,3 +42,17 @@ $xxlarge-only: '#{$screen} and (min-width:#{lower-bound($xxlarge-range)}) and (m
 $small-and-medium-only: '#{$screen} and (max-width: #{upper-bound($tablet-range)})';
 $medium-up-to-large: '#{$screen} and (min-width:#{lower-bound($tablet-range)}) and (max-width:#{upper-bound($desktop-range)})';
 $medium-up-to-xxlarge: '#{$screen} and (min-width:#{lower-bound($tablet-range)}) and (max-width:#{upper-bound($xlarge-range)})';
+
+$IE: 'all and (-ms-high-contrast: none), (-ms-high-contrast: active)';
+
+// Use to compensate for scroll-to offset
+// This is the height of the main nav from mobile up
+$tablet-nav-height: 120px;
+
+$gutter-width: ru(.5);
+
+// -12px added to take into account gutter that comes with Layout in each section
+$section-padding-mobile: rem-calc(24px - 12px);
+$section-padding-tablet: rem-calc(48px - 12px);
+$section-padding-desktop: rem-calc(64px - 12px);
```

Comment:
Can you do `rem-calc(24px - $gutter-width)` instead?

Reply:
👍 

Also, these can probably go into a variables file instead of the breakpoints. The `$IE` makes sense here, but, even though the others are related, they aren't really break points.

Reply:
`rem-calc(24px - $gutter-width)` failed the build, i think because of mixing units. I also tried re-defining `$gutter-width` with `rem-calc`, same issue.